### DISCUSSION TITLE
Submenu in Sidebar

### DISCUSSION THREAD

**Discussion by shahaab-ali** [saadeghi/daisyui#1061] [Q&A] [answered]
Hi, How to create collapse able submenu in sidebar with dasiyui?

**Comment by rhscjohn-dev** [comment]
Why  do you want to use the Collapse Component?   Wouldn't the Dropdown Component be a better fit.

**Reply by shahaab-ali** [reply]
Naah! I just want collaps able menu like bootstrap sidebar menu

**Comment by saadeghi** [comment, marked as answer]
See this https://github.com/saadeghi/daisyui/issues/896#issuecomment-1213595694

**Reply by shahaab-ali** [reply]
Thank you.

**Comment by robbins23** [comment]
Have created starter kit for dashboard using Daisy UI, do check. Submenu feature is there

[Daisy UI Dashboard Starter kit](https://github.com/srobbin01/tailwind-dashboard-template-dashwind)
